README.md

```diff
@@ -601,6 +605,63 @@ openssl req -x509 -nodes -days 365 -newkey rsa:2048 \
 cp tests/hostnames/hostname3.fullchain tests/hostnames/hostname3.chain
 ```
 
+## Select API Support
+
+MinIO instance supports SQL-style SELECT command and API support via its MinIO
+Client (`mc`) and language bindings.
+
+See [MinIO Select API Quickstart Guide](https://docs.min.io/docs/minio-select-api-quickstart-guide.html).
+
+As mentioned in the docs, by default it supports querying of CSV and JSON
+objects in uncompressed and compressed formats. No extra configuration is
+required.
+
+### Parquet Format
+
+The uncompressed [Parquet](https://en.wikipedia.org/wiki/Apache_Parquet) format
+is also supported but it not enabled by default. It can be enabled by setting
+the environment variable i.e. `MINIO_API_SELECT_PARQUET=on`.
+
+For development with [`docker-compose`](#using-docker-compose), it can be set
+like this:
+
+```yaml
+services:
+  # ...
+  minio:
+    image: minio
+    # ...
+    environment:
+      # ...
+      MINIO_API_SELECT_PARQUET: "on"
+```
+
+For development with [`docker-compose-gateway*`](#gateway-mode) , it can be
+enabled like this in the respective `docker-compose-gateway*.yaml` file:
+
+```yaml
+services:
+  # ...
+  minio-gateway:
+    image: minio
+    # ...
+    environment:
+      # ...
+      MINIO_API_SELECT_PARQUET: "on"
+```
```

Comment:
please allow to enable parquet from the .env file so we won't need to edit the docker-compose files

like we do for enabling cache:
https://github.com/CloudWebManage/cwm-worker-deployment-minio/blob/main/README.md#testing-the-cache-layer-locally-using-docker-compose